CHANGELOG.md

```diff
@@ -33,7 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   more dynamic than Ansible. The functionality was moved to utils GUI app
   which resides in the DBA Google drive.
 
-[Unreleased]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.2.0...HEAD
+[Unreleased]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.2.1...HEAD
 
 [v1.2.1]: https://github.com/CruGlobal/cru-ansible-modules/compare/v1.1.0...v1.2.1
```

Comment:
Noticed that this link should be `https://github.com/CruGlobal/cru-ansible-modules/compare/v1.2.0...v1.2.1`

That will open a page that compares just the changes between v1.2.0 and v1.2.1.